test_runner/regress/test_pageserver_api.py

```diff
@@ -42,6 +42,8 @@ def run_pageserver(args):
         "listen_http_addr",
         "pg_auth_type",
         "http_auth_type",
+        # TODO: only needed for NEON_PAGESERVER_PANIC_ON_UNSPECIFIED_COMPACTION_ALGORITHM in https://github.com/neondatabase/neon/pull/7748
```

Comment:
This is okay if intentional, but looks like it might be unintended?

Reply:
It's intentional and will be removed shortly by https://github.com/neondatabase/neon/pull/7766